### PR TITLE
Remove some multi-token support feature flags

### LINKED ISSUE #2264: Comment in payment deeplinks are not displayed in the description field
### Current behavior

The comment field in payment deeplinks is ignored and not displayed in the description field for confirming the transaction.

### Desired behavior

The comment field in payment deeplinks should be displayed and added to the transaction for tokens supporting transfers with comment (CELO, cUSD, cEUR, cBRL).

### Steps to Reproduce

1. Scan the following QR code:
download (1)
Or deeplink: celo://wallet/pay?address=0x4b371df8d05abd2954564b54faf10b8c8f1bc3a2&displayName=Example%20name&amount=9.50&comment=Burger%20with%20fries&token=cUSD&currencyCode=USD (sorry GitHub doesn't support celo:// links, you'll have to copy and paste that in a note to click it).
2. Notice the expected comment "Burger with fries" is not displayed in the transaction confirmation screen.

### Versions

Not sure exactly when this broke, but it can be reproduced with Valora v1.28 on both iOS and Android

### LINKED ISSUE #2278: Remove remote config and old code for multi token features
### Description
It has been a while since we've launched multi token support, it has been a success and there is no reason to believe that we will roll back to an app without multi token support. There are 3 remote config flags, `multiTokenShowHomeBalances`, `multiTokenUseSendFlow`, `multiTokenUseUpdatedFeed` which can be removed and the accompanying code can also be removed. This reduces our app size, app complexity, less code to maintain.

### AC
- [ ] remove the multi token remote config variables from firebase
- [ ] remove references of multi token remote config variables from the wallet project
- [ ] remove legacy code that will no longer be used

### PR DESCRIPTION
**Pull request by gnardini**
### Description

Reopened the PR from https://github.com/valora-inc/wallet/pull/2282 I messed up while fixing conflicts and GitHub closed it automatically :(

Removed a couple feature flags related to multi-token support. Most of the code is just this but in one file (`src/send/utils.ts`) there are more changes. There were two issues with deep links:
- The token currency was not being respected. See https://valora-app.slack.com/archives/C02G5197LAC/p1646939499924949
- The comment field was not being passed correctly to the `SendConfirmation` screen. See https://valora-app.slack.com/archives/C029Z1QMD7B/p1648476457612589

Using the deeplink `celo://wallet/pay?address=0xC0509a7dcc69a0B28c7Ca73feD2FF06b9d59E5b9&displayName=Kenneth%20Kangethe&currencyCode=KES&amount=0.1&comment=sending+kes:+0.1+to+My+kotani+wallet` I now get:
Screen Shot 2022-03-31 at 16 41 54

### Other changes

N/A

### Tested

Manually and with unit tests

### How others should test

Send deep links with different values and make sure they all work correctly.

### Related issues

- Fixes #2278
- Fixes #2264

### Backwards compatibility

N/A

### CODE REVIEW

File: packages/mobile/e2e/src/usecases/OffRamps.js [hunk 1]
```diff
@@ -161,11 +161,11 @@ export default offRamps = () => {
...
-        await expect(
-          element(by.text(`-${randomAmount} CELO`).withAncestor(by.id('TransactionList'))).atIndex(
-            0
-          )
-        ).toBeVisible()
+        const element = element(
+          by.text(`-${randomAmount} CELO`).withAncestor(by.id('TransactionList'))
+        ).atIndex(0)
+        await waitFor(element).toBeVisible().withTimeout(30000)
+        await expect(element).toBeVisible()
```
Comment (kathaypacific) [inline packages/mobile/e2e/src/usecases/OffRamps.js:168]: do these 2 assertions do the same thing?

**Review by kathaypacific — APPROVED** [review]
yay nice job!!! 🚀

**Comment by MuckT** [comment]
@gnardini I have #2311 working in parity with this PR and have some strange fails with e2e on that branch.

**Comment by MuckT** [comment]
@gnardini I've found a few bugs with this PR working to write the e2e tests.
1. Withdrawal to a CELO account are shown as additions

https://user-images.githubusercontent.com/26950305/163634186-d43f66a4-ba7b-46d0-800a-4d21ed911a7c.mov

2. Comments no longer appear in transactions

https://user-images.githubusercontent.com/26950305/163634337-9d81e48b-176e-45b3-ac13-3ce5ff3a17c5.mov

**Comment by MuckT** [comment]
@gnardini passing e2e tests for this PR are included in #2311!

**Comment by gnardini** [comment]
Closing since the changes here will be merged with https://github.com/valora-inc/wallet/pull/2311 and syncing both would be too hard

**Pull request closed without merging**